trunk/src/lsc/myloopdef.py

```diff
@@ -336,6 +336,10 @@ def run_psf(imglist, treshold=5, interactive=False, _fwhm='', show=False, redo=F
             cc=' --catalog '+catalog+' '
         else:
             cc=' '
+        if field is not None:
+            field_cmd = ' --field ' + field + ' '
+        else:
+            field_cmd = ' '
```

Comment:
This creates an issue while running the `lscpsf.py` command.
In `lscloop.py` `--field` is always initialized with an empty string. So, the if statement here will always run.